### DISCUSSION TITLE
Allow adding new languages for TranslationServer

### DISCUSSION THREAD

**Discussion by GameDungeon** [godotengine/godot-proposals#6246] [General Discussions]
The list of supported locales is not comprehensive, and I would see great benefit in being able to register additional language that are missing.

Also registering new languages allows for non conventional languages as well Eg. Minecraft's pirate language.

**Comment by timothyqiu** [comment]
Closing as a corresponding proposal is created: https://github.com/godotengine/godot-proposals/issues/6574
